Lib/glyphsLib/builder/instances.py

```diff
@@ -108,19 +108,25 @@ def _to_designspace_instance(self, instance):
             linked_style=instance.linkStyle,
         )
         ufo_instance.styleMapFamilyName = styleMapFamilyName
-        ufo_instance.styleMapStyleName = styleMapStyleName
+        if styleMapStyleName != "regular":
+            ufo_instance.styleMapStyleName = styleMapStyleName
 
     ufo_instance.name = " ".join(
         (ufo_instance.familyName or "", ufo_instance.styleName or "")
     )
 
     if self.minimize_glyphs_diffs:
-        ufo_instance.lib[EXPORT_KEY] = instance.active
+        if instance.active is not True:
+            ufo_instance.lib[EXPORT_KEY] = instance.active
         ufo_instance.lib[WEIGHT_KEY] = instance.weight
         ufo_instance.lib[WIDTH_KEY] = instance.width
 
-        ufo_instance.lib[INSTANCE_INTERPOLATIONS_KEY] = instance.instanceInterpolations
-        ufo_instance.lib[MANUAL_INTERPOLATION_KEY] = instance.manualInterpolation
+        if instance.instanceInterpolations:
+            ufo_instance.lib[
+                INSTANCE_INTERPOLATIONS_KEY
+            ] = instance.instanceInterpolations
+        if instance.manualInterpolation is not False:
```

Comment:
`is not False`? what's this?

Reply:
I think that's a 🧠💨. Something felt wrong but I couldn't place my finger on it.